lib/membrane_audio_mixer.ex

```diff
@@ -39,6 +44,15 @@ defmodule Membrane.AudioMixer do
                 Used when converting demand from buffers into bytes.
                 """,
                 default: 2048
+              ],
+              declipper: [
+                type: :boolean,
+                spec: boolean(),
+                description: """
+                If false, overflowed waves will be clipped to the max value.
+                If true, overflowed waves will be scaled down to the max value.
+                """,
+                default: false
```

Comment:
I'm open to discussion, but I think we can make clipping prevention a default behaviour